content/docs/building-apps/first-deposit.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: Fund the Account and make the First Deposit
+order: 65
+---
+
+In this section, we will go over the new user account creation flow between non-custodial wallets and anchors with SEP-24 and/or SEP-6 implementations. Before we dive into the flow, it’s important to understand how Stellar accounts are created in the first place. 
+
+The first step in creating a Stellar account is to generate a keypair. This keypair includes public and private keys. However, an account does not exist and does not warrant space on the ledger until it’s funded with the minimum balance of 1XLM by a [Create Account](../../start/list-of-operations.mdx#create-acccount) operation. This requirement was created to minimize unused accounts from bloating the ledger. For a more in-depth look at this topic, check out this [page](../tutorials/create-account.mdx).
+
+## Who creates the new user accounts? 
+
+When a new customer downloads the wallet application and goes through the deposit flow for the first time, their Stellar account can be created by either the user’s wallet application or the anchor facilitating the first deposit. This section describes the possible strategies and flows for you to consider. 
+
+### Option 1: The anchor creates and funds the Stellar account
+
+For this option, the wallet needs to allow users to initiate their first deposit without having to add an asset/establish a trustline. The wallet then prompts the user to add the trustline once funds are received by the anchor. The flow looks like this: 
+
+1. The wallet registers a new user and issues a keypair.
+2. The wallet initiates the first deposit on behalf of the user without requiring the user to add the asset/create the trustline.
+3. The anchor provides deposit instructions to the customer.
+4. The user transfers money from a bank account to the anchor’s bank account.
+5. Once the anchor receives the transfer, the anchor creates and funds the Stellar account for the customer. 
+
+*Note*: An anchor should always maintain a healthy amount of XLM in their distribution account to support new account creations. If doing so becomes unsustainable, it’s recommended that the anchor collaborates with wallets to determine a strategy based on the number of account creation requests. The recommended amount is 2XLM per user account creation (1XLM to meet the minimum balance requirement, and 1XLM for establishing trustlines and covering transaction fees).
+
+7. The wallet detects that the account has been created and a trustline must be established.
+8. The wallet prompts the user to add the asset/create the trustline.
+9. Finally, the anchor sends the deposit funds to the user’s Stellar account.
+
+![anchor creates account flow](../../web/assets/first-deposit-anchor-flow.png)
+
```

Comment:
Unfortunately, adding a paragraph like this in the middle of the list breaks numbering. The second part of the list starts at (1) again... Does it still fit okay at the end of the list, instead?

```suggestion
5. Once the anchor receives the transfer, the anchor creates and funds the Stellar account for the customer. 
6. The wallet detects that the account has been created and a trustline must be established.
7. The wallet prompts the user to add the asset/create the trustline.
8. Finally, the anchor sends the deposit funds to the user’s Stellar account.

**Note**: An anchor should always maintain a healthy amount of XLM in its distribution account to support new account creations. If doing so becomes unsustainable, it’s recommended that the anchor collaborates with wallets to determine a strategy based on the number of account creation requests. The recommended amount is 2XLM per user account creation (1XLM to meet the minimum balance requirement, and 1XLM for establishing trustlines and covering transaction fees).

![anchor creates account flow](../../web/assets/first-deposit-anchor-flow.png)

```